scripts/update-app-secrets.sh

```diff
@@ -8,12 +8,6 @@
 appName="TestApp"
 appPrefix="INT"
```

Comment:
We should get appPrefix from args

Reply:
We don't have `PROD` anymore, so argument is not needed.